packages/cardano-services/test/Asset/TypeOrmNftMetadataService.test.ts

```diff
@@ -0,0 +1,33 @@
+import { TypeOrmNftMetadataService, createDnsResolver, getConnectionConfig, getEntities } from '../../src';
+// import { handleAssetInfo } from '@cardano-sdk/util-dev/dist/cjs/mockProviders';
+import { logger, mockProviders } from '@cardano-sdk/util-dev';
+
+describe('TypeOrmNftMetadataService', () => {
+  let service: TypeOrmNftMetadataService;
+
+  beforeAll(async () => {
+    const dnsResolver = createDnsResolver({ factor: 1.1, maxRetryTime: 1000 }, logger);
+    const entities = getEntities(['nftMetadata', 'asset', 'block']);
+    const connectionConfig$ = getConnectionConfig(dnsResolver, 'test', 'Handle', {
+      postgresConnectionStringHandle: process.env.POSTGRES_CONNECTION_STRING_HANDLE!
+    });
+    service = new TypeOrmNftMetadataService({ connectionConfig$, entities, logger });
+    await service.initialize();
+    await service.start();
+  });
+
+  afterAll(async () => {
+    jest.restoreAllMocks();
+    await service.shutdown();
+  });
+
+  it('get an nft metadata when assetId and name is provided', async () => {
+    const nfMetadata = await service.getNftMetadata({
+      assetId: mockProviders.handleAssetInfo.assetId,
```

Comment:
This is not part of the interface, so it's confusing to have it here
```suggestion
```

Reply:
Fixed on 2c261b3